build.sbt

```diff
@@ -384,8 +385,8 @@ lazy val scaldingParquetFixtures = module("parquet-fixtures")
      )
    )
 
-lazy val scaldingParquet = module("parquet").settings(
-  libraryDependencies <++= (scalaVersion) { scalaVersion => Seq(
+lazy val scaldingParquetCascading = module("parquet-cascading").settings(
```

Comment:
ok, so the point here is that this target only depends on cascading + parquet, not scalding. Can we add a comment so that is not overlooked as to the purpose?